lib/client.js

```diff
@@ -30,6 +30,7 @@ const Logger = require('./logger');
 const { POOL_CONFIG_OPTIONS } = require('./constants');
 const ViewBuilder = require('./schema/viewbuilder.js');
 const ViewCompiler = require('./schema/viewcompiler.js');
+const { isPlainObject } = require('lodash');
```

Comment:
It's preferable to use `require('lodash/isPlainObject');` style, because that is tree-shaking friendly (for some users it is actually important)